<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EC2AMAZ-8HCHO4G" failures="1" tests="6" name="com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED" time="695.424" errors="0" timestamp="2024-01-22T18:32:21 UTC" skipped="0">
  <testcase classname="com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED" name="ingresar" time="5.703"/>
  <system-out/>
  <testcase classname="com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED" name="mdrEntrar" time="16.466"/>
  <system-out/>
  <testcase classname="com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED" name="criar" time="262.085"/>
  <system-out/>
  <testcase classname="com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED" name="criarSPED" time="105.367"/>
  <system-out/>
  <testcase classname="com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED" name="executar" time="221.901"/>
  <system-out/>
  <testcase classname="com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED" name="excluir" time="83.902">
    <failure message="O registro n�o foi eliminado com sucesso expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: O registro n�o foi eliminado com sucesso expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED.excluir(ExecucacoMapeamentoDoSPED.java:81)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- excluir -->
  <system-out/>
</testsuite> <!-- com.timp.test.DFG.Leiautes.ExecucacoMapeamentoDoSPED -->
